dotcom-rendering/src/components/Masthead/Masthead.tsx

```diff
@@ -51,7 +52,7 @@ export const Masthead = ({
 	idApiUrl,
 	contributionsServiceUrl,
 	showSubNav = true,
-	isImmersive,
+	showSlimNav,
```

Comment:
The majority of pages have slimNav set to false - is it worth setting that as a default value here?